doc/internal/global_seq_diag/erc20.puml

```diff
@@ -0,0 +1,48 @@
+@startuml
+autonumber
+
+skinparam roundcorner 20
+
+participant "Cape Contract" as C
+participant "Ethereum Query Service" as EQS
+participant "Alice's Wallet" as W_A
+participant "Bob's Wallet" as W_B
+participant "Relayer" as R
+participant "Freezer" as F
+participant "Auditor" as A
+participant "Ethereum Wallet Eve" as ETH_WX
+
+group Conversion between ERC20 and CAP assets
+
+    group Sponsor
+        W_A -> C: **sponsorCapeAsset**(USDC, asset_def)
```

Comment:
`sponsorCapeAsset`, not `sponsor_cape_asset`? This seems to be the only exception to the naming convention.

Reply:
Thanks! Fixed in 354fadfcd70676071a78ee3f2920e6c2cfd113bd.